torchrl/objectives/redq.py

```diff
@@ -253,6 +257,7 @@ def __init__(
         gSDE: bool = False,
         gamma: float = None,
         priority_key: str = None,
+        separate_losses: bool = True,
```

Comment:
I had to set redq defaults to true, otherwise the other existing tests failed.

Reply:
If I remove the REDQ tests and make them more lightweight, like the other tests I did (only focused on checking the gradients), I can change the defaults to False here and all tests pass. I will make a commit with this change.